### PR TITLE
Add seed mock meter flag for development.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -4,3 +4,6 @@
 import {config} from 'bedrock';
 
 config.meter = {};
+
+// seed bedrock-meter with mock meters for development
+config.meter.seedMockData = false;
```
Comment (dlongley) [inline lib/config.js]: Can we call this `addMockData` instead? `seed` confused me ... especially given the use of `seed` to mean a number of different things (it took a while to even consider it a verb here). Also, do we want it to be `addMockMeters` to be more specific?

Comment (dlongley) [reply lib/config.js]: Do we want to default this to `true` and require it to be changed via overrides? Won't devs want it to default to true? I'm not sure about use cases here.

**Review by dlongley — APPROVED** [review]
Approving but see comments.

File: lib/config.js (hunk 1 above)
Comment (gannan08) [reply lib/config.js]: We can change it to `addMockMeters` and default to `true`. This change will require us to enable config overrides.

**Pull request merged**